src/test/java/koval/myfit/GoogleFitTest.java

```diff
@@ -60,4 +65,80 @@ public void compareListOfTopics() throws InterruptedException {
                 "[ HOME PAGE / actualPlaylistTitlesList ] Actual list of Playlist titles is not equals to expected list!");
 
     }
+
+
+    @Test()
+    @MethodOwner(owner = "koval")
+    @TestLabel(name = "feature", value = {"mobile", "regression"})
+    public void addActivity() throws InterruptedException, ParseException {
+
+        HomePageBase homePageBase = initPage(getDriver(), HomePageBase.class);
+        Assert.assertTrue(homePageBase.isPageOpened(), "[ HOME PAGE ] Home page is not opened!");
+
+        TimeUnit.SECONDS.sleep(TIMEOUT_FIVE);
```

Comment:
move this part of code and use waitUntil(ExpectedConditions.parametr_you_need_to_OfElementLocated(xpath.getBy()), FIFTEEN);
inside of methods